<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image/icon.qrc">
    <normaloff>:/系统_system.png</normaloff>:/系统_system.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="btnStudentInfo">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>学员信息</string>
         </property>
         <property name="icon">
          <iconset resource="image/icon.qrc">
           <normaloff>:/书籍_bookshelf.png</normaloff>:/书籍_bookshelf.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSchedule">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>课程安排</string>
         </property>
         <property name="icon">
          <iconset resource="image/icon.qrc">
           <normaloff>:/日程表_schedule.png</normaloff>:/日程表_schedule.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnFinance">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>财务</string>
         </property>
         <property name="icon">
          <iconset resource="image/icon.qrc">
           <normaloff>:/理财_financing.png</normaloff>:/理财_financing.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnHonor">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>荣誉墙</string>
         </property>
         <property name="icon">
          <iconset resource="image/icon.qrc">
           <normaloff>:/长城_great-wall.png</normaloff>:/长城_great-wall.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSystemSetting">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>系统设置</string>
         </property>
         <property name="icon">
          <iconset resource="image/icon.qrc">
           <normaloff>:/设置_setting-one.png</normaloff>:/设置_setting-one.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>129</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="StudentInfoWidget" name="pageStudentInfo"/>
      <widget class="QWidget" name="pageSchedule">
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>190</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>pageSchedule</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageFinance">
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>240</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>pageFinance</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageHonor">
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>240</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>pageHonor</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageSystemSetting">
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>250</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>pageSystemSetting</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StudentInfoWidget</class>
   <extends>QWidget</extends>
   <header>studentinfowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image/icon.qrc"/>
 </resources>
 <connections/>
</ui>
